pkgs/top-level/aliases.nix

```diff
@@ -1410,6 +1410,7 @@ mapAliases ({
   qt514 = throw "Qt 5 versions prior to 5.15 are no longer supported upstream and have been removed";  # Added 2022-11-24
   qt515 = qt5;  # Added 2022-11-24
   qt5ct = libsForQt5.qt5ct; # Added 2021-12-27
+  qt6ct = qt6Packages.qt6ct; # Added 2023-03-07
```

Comment:
New packages should not immediately get an alias.

Reply:
Is this guideline documented somewhere? What is the difference between a nested package with an alias and the same package in the top level?

I am comparing this to `prismlauncher` and `prismlauncher-qt5` which are both top-level packages that use `qt6Packages.callPackage` and `libsForQt5.callPackage` respectively.

Reply:
I don't know either of your questions, aliases should only be added if packages are renamed to prevent evaluation errors for end users. Adding aliases for completeness is not done. Aliases are not allowed to be used in nixpkgs.